e2e/js_image/README.md

```diff
@@ -0,0 +1,36 @@
+# An example/e2e for rules_js + rules_docker
+
+`js_image_layer.bzl` helper contains some logic to make rules_docker, pkg_tar and rules_js work together.
+
+## Fine-grained layering
+
+`js_image_layer` is a macro that yields two tar files `:<name>/app.tar` and `:<name>/node_modules.tar` which helps to speed things up by splitting files into two layers. While `app.tar` contains your code, `node_modules.tar` contains your third-party dependencies.
```

Comment:
Try to avoid language like "your", and make technical documentation less personal.

```suggestion
`js_image_layer` is a macro that yields two tar files `:<name>/app.tar` and `:<name>/node_modules.tar`. This increases performance by splitting files into two layers. While `app.tar` contains first-party sources, `node_modules.tar` contains all third-party dependencies.
```

Perhaps add _why_ this helps improve perf?